.travis-build.sh

```diff
@@ -30,12 +33,16 @@ set -o xtrace
 export SHELLOPTS
 
 
-./.travis-build-without-test.sh
-# The above command builds the JDK, so there is no need for a subsequent
-# command to rebuild it again.
+./.travis-build-without-test.sh $JDKVER
+# The above command downloads the JDK, so there is no need for a subsequent
+# command to build it except to test building it.
 
 set -e
 
+if [[ "${GROUP}" == "all-tests" || "${GROUP}" == "all" ]]; then
+  (cd checker && ant all-tests-nobuildjdk)
+fi
+
 if [[ "${GROUP}" == "junit" || "${GROUP}" == "all" ]]; then
```

Comment:
Is it still useful to have the junit and nonjunit tests?  Before this pull request, all the permitted targets were disjoint, but not any longer and it might be good to document that (or remove the subtasks to avoid confusion over why they aren't being called in the test matrix).